motor_info:
        min_pwm: 0.6 # 1200 [ms]
        max_pwm: 0.925 # 1850 [ms]
        min_thrust: 0.0 #  [N]
        force_landing_thrust: 5.0 #[N]
        m_f_rate: -0.0134
        pwm_conversion_mode: 1 # Polynominal Mode
        vel_ref_num: 5
        ref1:
                voltage: 26.2
                max_thrust: 24.4 # N
                polynominal4: 2.6374697    # x 10^4
                polynominal3: -13.588596   # x 10^3
                polynominal2: 19.442409    # x 10^2
                polynominal1: 9.5308610    # x 10
                polynominal0: 57.541709    # x 1
        ref2:
                voltage: 25.2
                max_thrust: 23.2 # N
                polynominal4: 2.8932558   # x 10^4
                polynominal3: -14.738371  # x 10^3
                polynominal2: 20.857811   # x 10^2
                polynominal1: 9.8227477   # x 10
                polynominal0: 57.705457   # x 1
        ref3:
                voltage: 24.2
                max_thrust: 21.6 # N
                polynominal4: 4.0569161   # x 10^4
                polynominal3: -19.001922  # x 10^3
                polynominal2: 24.781381   # x 10^2
                polynominal1: 10.423286   # x 10
                polynominal0: 57.660644   # x 1
        ref4:
                voltage: 23.2
                max_thrust: 20.3 # N
                polynominal4: 4.7340846    # x 10^4
                polynominal3: -21.661921  # x 10^3
                polynominal2: 27.980902   # x 10^2
                polynominal1: 9.8488811   # x 10
                polynominal0: 58.125480   # x 1
        ref5:
                voltage: 22.2
                max_thrust: 18.8 # N
                polynominal4: 6.9273301    # x 10^4
                polynominal3: -29.412858  # x 10^3
                polynominal2: 35.616376   # x 10^2
                polynominal1: 9.1384463   # x 10
                polynominal0: 58.265317   # x 1
